content/en/docs/pipelines/installation/localcluster-deployment.md

```diff
@@ -283,12 +283,20 @@ line. (For Kubeflow and other component support, check [K3ai's
 website](https://kf5ai.gitbook.io/k3ai/#components-of-k-3-ai) for updates.) To
 install Kubeflow Pipelines using K3ai, run the following command:
```

Comment:
Since there're two options for CPU and GPU support, let's make this plural:

```suggestion
install Kubeflow Pipelines using K3ai, run the following commands:
```